src/nodejs_bindings.cpp

```diff
@@ -33,12 +33,26 @@ NAN_MODULE_INIT(Annotator::Init)
 
 NAN_METHOD(Annotator::New)
 {
-    if (info.Length() != 0)
-        return Nan::ThrowTypeError("No types expected");
+    bool coordinates = false;
+    if (info.Length() != 0) {
+        if (!info[0]->IsObject())
+            return Nan::ThrowTypeError("Options should be an object");
+        const auto options = info[0].As<v8::Object>();
+        auto tryParseCoordinates = Nan::Get(options, Nan::New("coordinates").ToLocalChecked());
+        if (!tryParseCoordinates.IsEmpty())
+        {
+            auto parsedCoords = tryParseCoordinates.ToLocalChecked();
+            if (parsedCoords->IsBoolean())
+            {
+                coordinates = parsedCoords->BooleanValue();
+            } else return Nan::ThrowTypeError("Coordinates value should be a boolean"); // todo check what kind of error to throw
+        }
```

Comment:
I think there's an else branch missing here for the IsEmpty check - if Get fails the object key isn't there

Reply:
Hmm, I think it would be OK to let the default behavior here stay, which is that we would effectively ignore an empty options object.

Reply:
But it also allows for the mistake where the user accidentally spells the option's key wrong

```js
new Annotator({'coordinate': true})
new Annotator({'coordinates': true})
```

if the key really should be `coordinates` and is `false` by default the first line ^ does not set it to `true` but also does not error out.